src/snmp/Forwarder.cc

```diff
@@ -100,7 +100,7 @@ Snmp::SendResponse(unsigned int requestId, const Pdu& pdu)
         response.pdu = static_cast<Pdu&>(*response_pdu);
         snmp_free_pdu(response_pdu);
     } catch (const std::exception& e) {
-        debugs(49, DBG_CRITICAL, HERE << e.what());
+        debugs(49, DBG_CRITICAL, e.what());
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.